modules/ROOT/partials/configuration/tinycomments_fetch.adoc

```diff
@@ -0,0 +1,78 @@
+[[tinycomments_fetch]]
+== `+tinycomments_fetch+`
+
+The {pluginname} plugin uses the `+tinycomments_fetch+` callback function retrieves multiple conversations based on their unique identifiers.
+
+The conventional conversation object structure that should be returned via the `+done+` callback is as follows:
+
+The `+tinycomments_fetch+` function is passed a (`+req+`) object as the first parameter, which contains the following key-value pair:
+
+* `+conversationUids+`: An array of strings representing the unique identifiers of the conversations to fetch.
+
+The `+done+` callback should accept the following object:
+
+[source,js]
+----
+{
+  conversations: {
+    [conversationUid: string]: {
+      uid: string,
+      comments: [
+        {
+          uid: string,
+          author: string,
+          authorName?: string,
+          authorAvatar?: string,
+          content: string,
+          createdAt: string, // ISO 8601 date string
+          modifiedAt: string // ISO 8601 date string
+        },
+        // ... more comments
+      ]
+    },
+    // ... more conversations
+  }
+}
+----
+
+.For example:
+[source,js]
+----
+const fetch_comments = (conversationUids, done, fail) => {
+  fetch(`https://api.example/conversations`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ uids: conversationUids }),
+  })
+    .then(response => {
+      if (!response.ok) {
+        throw new Error('Network response was not ok');
+      }
+      return response.json();
+    })
+    .then((data) => {
+      console.log(`Lookup success ${conversationUids}`, data);
+      done(data);
+    })
+    .catch((err) => {
+      console.error(`Lookup failure ${conversationUids}`, err);
+      fail(err);
+    });
+};
```

Comment:
Please update this example as well

Reply:
Updated `tinycomments_fetch` example to match demo implementation, see [this comment](https://github.com/tinymce/tinymce-docs/pull/3458/commits/2e3f04df39bf985d1d95d1aece63985f3d7deeff)